groups/views.py

```diff
@@ -266,3 +282,50 @@ def delete(self, request, *args, **kwargs):
 
     def get_success_url(self):
         return self.comment.get_absolute_url()
+
+
+class CommentPostByEmail(CommentEmailMixin, View):
+    """
+    Receive comments posted by email and create them in the database.
+
+    This view is intended to be linked to an endpoint that provides an URL kwarg
+    'uuid'.  This matches up to an EmailUUID object that stores the discussion and user
+    being used.
+    """
+    def dispatch(self, request, *args, **kwargs):
+        self.uuid = kwargs.pop('uuid')
+        return super(CommentPostByEmail, self).dispatch(request, *args, **kwargs)
+
+    @staticmethod
+    def get_uuid_data(uuid):
```

Comment:
It could, although it is only used by this class and would be more difficult to find and follow without it.